synapse/storage/engines/__init__.py

```diff
@@ -18,7 +18,7 @@
 
 from ._base import IncorrectDatabaseSetup
 from .postgres import PostgresEngine
-from .sqlite3 import Sqlite3Engine
```

Comment:
Why the rename of the file? It seems odd when we haven't renamed the class.

Reply:
because it shadows a global package :/

Reply:
Ooooooh, fair :+1: